tests/epyccel/test_epyccel_generators.py

```diff
@@ -5,6 +5,9 @@
 
 from pyccel.epyccel import epyccel
 
+RTOL = 2e-14
+ATOL = 1e-15
+
```

Comment:
I suspect that the tests in this module should not introduce any roundoff. We usually start from a random sequence of small integers, convert it to float, and then apply simple operators.